### DISCUSSION TITLE
is the project still alive

### DISCUSSION THREAD

**Discussion by MatiasKottwitz** [trunkmaster/nextspace#435] [General]
Good, how are you? is the project still alive? I think the idea of renewing the NeXT STEP environment is great, I followed it very closely but after the last problems reported here it seems to have stopped. Some updates would be great. regards

**Comment by FuryOusFrank** [comment]
Hi. Sergii Stoian lives in Ukraine. As we can imagine, his priorities shifted with the Russian invasion.

**Comment by MatiasKottwitz** [comment]
Hello! , you are right, I apologize, although I was just asking out of curiosity because it is a good project

**Comment by biggers** [comment]
Time to fork this project, with a new maintainer?   Eager to see it improve and grow.  Thanks.

**Comment by lproven** [comment]
> Time to fork this project, with a new maintainer? Eager to see it improve and grow. Thanks.

Not a fork but a related project:

https://onflapp.github.io/gs-desktop/index.html

https://github.com/onflapp/gs-desktop

**Comment by trunkmaster** [comment]
Hello everybody!

I'm physically OK, but at the moment have no inspiration to spend time to project. The reasons: war, continuous air alerts, fear to be grabbed into the army anytime. As a result my future is uncertian and it's quite exausting.

Anyway, there's no need to fork the project: I can grant developer rights to anyone who dare to continue it. Just let me know who have enough skills to continue project development. You may find task list in [Projects ](https://github.com/trunkmaster/nextspace/projects)section.

I hope I will be able to return to project development somewhere in the future.

**Comment by roblillack** [comment]
@trunkmaster: What a relief! Wish you all the best.

**Comment by pramsky** [comment]
> Hello everybody!
> 
> I'm physically OK, but at the moment have no inspiration to spend time to project. The reasons: war, continuous air alerts, fear to be grabbed into the army anytime. As a result my future is uncertian and it's quite exausting.
> 
> Anyway, there's no need to fork the project: I can grant developer rights to anyone who dare to continue it. Just let me know who have enough skills to continue project development. You may find task list in [Projects ](https://github.com/trunkmaster/nextspace/projects)section.
> 
> I hope I will be able to return to project development somewhere in the future.

Glad to hear you are ok.

**Comment by mbert** [comment]
> Hello everybody!
> 
> I'm physically OK, but at the moment have no inspiration to spend time to project. The reasons: war, continuous air alerts, fear to be grabbed into the army anytime. As a result my future is uncertian and it's quite exausting.
> 
> Anyway, there's no need to fork the project: I can grant developer rights to anyone who dare to continue it. Just let me know who have enough skills to continue project development. You may find task list in [Projects ](https://github.com/trunkmaster/nextspace/projects)section.
> 
> I hope I will be able to return to project development somewhere in the future.

Good to see that you are OK. We support you from wherever we are. Все буде Україна!

**Comment by coderamen666** [comment]
Slava Ukraini!
